postgresfriends.json

```diff
@@ -23,7 +23,7 @@
 	"Paul Ramsey": {
 		"Blog": ["https://blog.cleverelephant.ca/"],
 		"Mastodon": ["https://mastodon.social/@pwramsey"],
-		"Twitter": ["https://twitter.com/pramsey"],
+		"Twitter": ["https://twitter.com/pwramsey"],
```

Comment:
Ha, I added plenty of sanity checks - but can't automatically verify if the URL is the correct URL ;-)
Thanks for catching this!